macos/FluentUI/DatePicker/CalendarView.swift

```diff
@@ -58,7 +58,7 @@ class CalendarView: NSView {
 		}
 		
 		// Accessibility
-		setAccessibilityElement(!DatePickerView.accessibilityTemporarilyRestricted)
+		setAccessibilityElement(true)
```

Comment:
I'm assuming the default value is `false` for everywhere we're explicitly setting to `true`?

Reply:
> By default, NSView and its subclasses set this value to NO; however, if your NSView subclass adopts one of the accessibility protocols, the system changes the default value to YES.

Reply:
Yeah from the snippets in the review it wasn't immediately obvious if we were subclassing NSControls or just pure NSViews.